data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/main/Main.java

```diff
@@ -70,79 +58,74 @@
    * @param args command line arguments.
    */
   public static void main(final String[] args) throws IOException {
+    DispatcherEnv env = new DispatcherEnv(System::getenv);
+
+    OpenTelemetrySdk openTelemetry = TracingConfig.fromDir(env.getConfigTracingPath()).setup();
+
     // HACK HACK HACK
     // maven-shade-plugin doesn't include the LogstashEncoder class, neither by specifying the
     // dependency with scope `provided` nor `runtime`, and adding include rules to
     // maven-shade-plugin.
     // Instantiating an Encoder here we force it to include the class.
+    // TODO this requires some fix in our maven-shade-plugin usage
```

Comment:
is there a bug on the `maven-shade-plugin` or somewhere else for tracking reasons?

Reply:
I don't know, I never investigated this... Maybe it's a runtime classpath scanning that recognizes this encoder, hence the issue that maven-shade-plugin doesn't know how to load it. But I think we can manually instruct the plugin to load this class anyway

Reply:
Can we create an issue?

Reply:
Done :+1: